sw/device/sca/lib/simple_serial.h

```diff
@@ -32,6 +33,17 @@
     }                                                \
   } while (false)
 
+/**
+ * Sends an error message over UART if DIF does not return kDifOk.
+ */
+#define SS_CHECK_DIF_OK(dif_call)                    \
+  do {                                               \
+    if (dif_call != kDifOk) {                        \
+      simple_serial_send_status(kSimpleSerialError); \
```

Comment:
Could we augment `test_status_set` to satisfy this requirement, rather than have to maintain an alternate implementation that almost does the same thing? @moidx

Reply:
This reporting mechanism conforms to a serial protocol we use for SCA analysis. Lets keep them separate for now.

Reply:
File a followup bug?